hasher-matcher-actioner/hmalib/common/aws_dataclass.py

```diff
@@ -33,6 +33,11 @@ class Item(HasAWSSerialization):
 
 T = t.TypeVar("T")
 
+# dynamodb does not allow sets to be empty. Use this as the only member when
+# writing an empty set. When retrieving a set, remove this if it is the only
```

Comment:
nit: current impl seem to always remove this value this if it is in the set, regardless.

Reply:
David flagged an easier check that performs better. That will also take care of this issue. Thanks!